Documentation/Storage-Configuration/Backup-And-Recovery

```diff
@@ -0,0 +1 @@
+../../ee/docs/Backup-And-Recovery
```

Comment:
The folder `Backup-And-Recovery` doesn't exist in the `ee/docs` folder, let's move both docs to that folder and keep the link here

Reply:
yep, good catch, updated, thanks!